src/Api/Endpoints/Projects.php

```diff
@@ -82,78 +81,71 @@ protected function supportedVersions() : array
      */
     public function handle(ServerRequestInterface $request) : ResponseInterface
     {
-        // FIXME: Validate permissions.
+        $user = $request->getAttribute('user');
+        if (!$this->_hasAccess($user)) {
+            return new \LORIS\Http\Response\Unauthorized();
```

Comment:
I think this should be Forbidden, not Unauthorized

Reply:
Debatable. In this case, `_hasAccess` return false when the user is not logged in. Forbidden response are usually sent when the user is authenticated but can't access the resource/service.

Reply:
Got it, I didn't realize that this can the case for an unauthenticated user. You're correct then and 401 is appropriate